index.html

```diff
@@ -787,8 +787,16 @@ <h2>Links</h2>
 
       <p>
         <span class="rfc2119-assertion" id="common-constraints-links-1">The following keywords are defined for the
-          HTTP profiles and MUST be supported by all consumers.</span>
+          HTTP profiles and MAY be present in compliant TDs.</span>
+	<span class="rfc2119-assertion" id="common-constraints-links-1a">Compliant consumers that have the capabilities 
+	  to handle these relation types SHOULD treat the link targets appropriately.</span>
```

Comment:
I'm not sure why we force the consumer to do something with the relation type. E.g., if a user interface is implemented in such a way that it can also display an icon, it would be great to use this possibility provided in the links. However, if the UI is not supporting displaying an icon that should be also fine.

Reply:
Agreeing with the comment above. Also, "handling appropriately" has no real meaning to be useful as an assertion.

Reply:
I have improved the language, please review the new text.